src/components/VaccinationsTable/utils.ts

```diff
@@ -1,53 +1,68 @@
-import regions, { State } from 'common/regions';
-import sortBy from 'lodash/sortBy';
+import regions, { Region } from 'common/regions';
+import orderBy from 'lodash/orderBy';
 import take from 'lodash/take';
 import takeRight from 'lodash/takeRight';
+import { MapView } from 'screens/HomePage/HomePage';
 import { LocationSummariesByFIPS } from 'common/location_summaries';
+import { Metric } from 'common/metricEnum';
 
-export interface StateWithVaccinationMetrics {
-  stateName: string;
-  vaccinationsInitiated: number | null;
-  vaccinationsCompleted: number | null;
-}
-
-export interface StateWithVaccinationMetricsAndRank {
+export interface RegionVaccinationInfo {
+  regionName: string;
+  vaccinationsInitiated: number;
+  vaccinationsCompleted: number;
   rank: number;
 }
 
-function getStatesSortedByVaccinationsCompleted(): StateWithVaccinationMetricsAndRank[] {
-  const statesWithVaccinationMetrics = regions.states.map(
-    (state: State, i: number) => {
-      const summaryForFips = LocationSummariesByFIPS[state.fipsCode];
-      return {
-        stateName: state.name,
-        vaccinationsInitiated: summaryForFips.metrics[6]?.value ?? null,
-        vaccinationsCompleted: summaryForFips.vc ?? null,
-      };
-    },
+function getRegionsSortedByVaccinationsInitiated(
+  regionScope: MapView,
+): RegionVaccinationInfo[] {
+  const regionsToSort =
+    regionScope === MapView.COUNTIES ? regions.counties : regions.states;
+
+  const regionsWithVaccinationMetrics = regionsToSort.map((region: Region) => {
+    const summaryForFips = LocationSummariesByFIPS[region.fipsCode];
+    return {
+      regionName: region.name,
```

Comment:
For counties we probably want to include the state abbreviation since I have no idea what state most counties are in. 😁   Can we just do whatever the compare table does for names?

Reply:
oh true